src/go/kreq-gen/CMakeLists.txt

```diff
@@ -1,3 +1,9 @@
+find_program(GO_EXECUTABLE go)
+set(CMAKE_GO_BINARY "${GO_EXECUTABLE}" CACHE PATH "path to go executable")
```

Comment:
also, i'd suggest avoid adding "CMAKE_" prefix to variables not set by cmake. if it's fine not preserving `CMAKE_GO_BINARY`, i would be happy to replace it with `GO_EXECUTABLE`.